light-client/eth-light-client/rust-toolchain.toml

```diff
@@ -0,0 +1,10 @@
+[toolchain]
```

Comment:
do we really need it?

Reply:
I agree, we already  have `rust-toolchain.toml` at the top level...

Reply:
Remove it.